extensions/github/src/publish.ts

```diff
@@ -66,11 +58,49 @@ export async function publishRepository(gitAPI: GitAPI, repository?: Repository)
 		owner = user.data.login;
 	} catch (e) {
 		// User has cancelled sign in
-		quickpick.dispose();
 		return;
 	}
 
-	quickpick.busy = false;
+	let orgChoice: string = owner;
+	// TODO: While this api call is made, there isn't anything displayed to the
+	// user to show progress.  Showing either picker first risks having to swap
+	// them after the api call is finished and possibly causing confusion.
```

Comment:
We should just show a _loading_ quick pick and update it accordingly once we have the answer, right?